src/services/campaigns/campaigns.hooks.js

```diff
@@ -116,6 +117,7 @@ module.exports = {
       checkReviewer(),
       checkOwner(),
       sanitizeHtml('description'),
+      createCampaignSlug(),
```

Comment:
@mdehghani I reckon we should regenerate slug after title is changed, what do you think?